timezone_utils/forms.py

```diff
@@ -8,8 +8,11 @@
 # Django
 from django.core.exceptions import ValidationError
 from django.forms import CharField
-from django.utils.encoding import force_text
-from django.utils.translation import ugettext_lazy as _
+try:
+    from django.utils.encoding import force_str as force_text
+except ImportError:
+    from django.utils.encoding import force_text
```

Comment:
[memo]
force_text was removed.
https://docs.djangoproject.com/en/4.0/releases/4.0/#features-removed-in-4-0